src/managers/quorum.py

```diff
@@ -176,19 +177,22 @@ def update_acls(self) -> None:
 
             # FIXME: data-platform-libs should handle this when it's implemented
             if client.chroot:
-                requested_chroots.add(client.chroot)
+                if event and client.relation and client.relation.id == event.relation.id:
+                    continue  # skip broken chroots, so they're removed
```

Comment:
Waiting on the outcome of [same conversation for zk-vm](https://github.com/canonical/zookeeper-operator/pull/108#discussion_r1466357419)